src/nodes/fmod_event_emitter_2d.cpp

```diff
@@ -0,0 +1,194 @@
+//
+// Created by bitbrain on 2022-08-14.
+//
+
+#include "fmod_event_emitter_2d.h"
+#include "../godot_fmod.h"
+
+using namespace godot;
+
+const String EVENT_PREFIX = "event:/";
+const int UNDEFINED = -1;
+const int NOTIFICATION_PAUSED = 14;
```

Comment:
I tried to somehow access the constant from Godot itself but couldn't find a way to do so. Apparently, it sits within `Node` but the godot_cpp variant does not contain such a constant...

Reply:
Including the Node.hpp header and using Node::NOTIFICATION_PAUSED doesn't work?

Reply:
nope, it wouldn't like it. Also checked `global_constants.hpp` which also did not contain it..